deploy/helm/sumologic/upgrade-2.0.0.sh

```diff
@@ -657,12 +657,17 @@ function migrate_fluent_bit() {
     yq w -i "${TEMP_FILE}" 'fluent-bit.service.labels."sumologic.com/scrape"' --style double true
   fi
 
-  yq w -i "${TEMP_FILE}" 'fluent-bit.extraVolumeMounts[+].mountPath' '/tail-db'
-  yq w -i "${TEMP_FILE}" 'fluent-bit.extraVolumeMounts[0].name' 'tail-db'
+  local EXTRA_VOLUMES
+  readonly EXTRA_VOLUMES="$(yq r "${TEMP_FILE}" -- 'fluent-bit.extraVolumes')"
+  if [[ -n "${EXTRA_VOLUMES}" ]]; then
+    yq w -i "${TEMP_FILE}" 'fluent-bit.extraVolumes' --from <(echo "${EXTRA_VOLUMES}")
+  fi
 
-  yq w -i "${TEMP_FILE}" 'fluent-bit.extraVolumes[+].hostPath.path' '/var/lib/fluent-bit'
-  yq w -i "${TEMP_FILE}" 'fluent-bit.extraVolumes[0].hostPath.type' 'DirectoryOrCreate'
-  yq w -i "${TEMP_FILE}" 'fluent-bit.extraVolumes[0].name' 'tail-db'
+  local EXTRA_VOLUME_MOUNTS
+  readonly EXTRA_VOLUME_MOUNTS="$(yq r "${TEMP_FILE}" -- 'fluent-bit.extraVolumeMounts')"
+  if [[ -n "${EXTRA_VOLUME_MOUNTS}" ]]; then
+    yq w -i "${TEMP_FILE}" 'fluent-bit.extraVolumeMounts' --from <(echo "${EXTRA_VOLUME_MOUNTS}")
+  fi
```

Comment:
If I understand correctly, you copy `fluent-bit.extraVolumeMounts` from `TEMP_FILE` as `fluent-bit.extraVolumeMounts` to `TEMP_FILE`

Reply:
Let's pretend this didn't happen. PTAL now